src/ZODB/tests/util.py

```diff
@@ -15,6 +15,7 @@
 """
 from ZODB.MappingStorage import DB
 
+import sys
```

Comment:
Can you move between `re` & `tempfile` so it remains somewhat sorted?

Reply:
ok